promlog/flag/flag.go

```diff
@@ -35,9 +35,11 @@ const FormatFlagHelp = "Output format of log messages. One of: [logfmt, json]"
 // AddFlags adds the flags used by this package to the Kingpin application.
 // To use the default Kingpin application, call AddFlags(kingpin.CommandLine)
 func AddFlags(a *kingpin.Application, config *promlog.Config) {
+	config.Level = new(promlog.AllowedLevel)
```

Comment:
`SetValue` needs a pointer to those flags (since the `Set` method is a pointer receiver), but I can also just do `config.Level = &promlog.AllowedLevel{}` if that's the preferred style.

Reply:
That'd be more usual

Reply:
Makes sense; changed it to be the more explicit declaration/initialization